Atomic/atomic.py

```diff
@@ -645,36 +645,62 @@ def gen_cmd(self, cargs):
             args.append(c)
         return " ".join(args)
 
+    def get_fq_name(self, image_info):
+        if len(image_info['RepoTags']) > 1:
+            raise ValueError("\n{} is tagged with multiple repositories. "
+                             "Please use a repository name instead.\n".format(self.image))
+        else:
```

Comment:
keep in mind the our docker fork stores the image already fully qualified (registry+name+tag). But it could happen to build locally for instance. If info takes a switch I don't believe we should ignore the switch, just fail